src/components/MainSlider/index.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import styled from 'styled-components';
+import Slider from 'react-slick';
+
+import SliderArrow from '../../ui-kit/components/SliderArrow';
+
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+
+const SliderWrapper = styled.div`
```

Comment:
again we could move styles into a separate file

Reply:
Done